src/serverless/app.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
```

Comment:
can `src/serverless` be moved to `src/standalone/serverless`? Since it is also an example of standalone usage?

Reply:
also is it possible to not rewrite this entire file, but reuse `src/standalone/index.js` instead?

Reply:
@ifedapoolarewaju can you check is it ok?

Reply:
yes, it's ok for the first issue, how about the second one?

> also is it possible to not rewrite this entire file, but reuse src/standalone/index.js instead?

Reply:
Yeah I tried that a couple of times it works locally but not on Lambda .. maybe I'm missing some config.
You can save up some lines on the main file, but you have to change the way the folders are structured.
```
// handler.js

const awsServerlessExpress = require('aws-serverless-express')
const awsServerlessExpressMiddleware = require('aws-serverless-express/middleware')

const { app } = require('../index')

app.use(awsServerlessExpressMiddleware.eventContext())

const server = awsServerlessExpress.createServer(app)

exports.uppy = (event, context) =>awsServerlessExpress.proxy(server, event, context)
```
Anyway when I get to check more about `sls` and `lambda` configs I'll upload another version.

Reply:
@dcflow alright, I'll merge this for now, and create an issue to refactor it so we don't forget about it. Thank you for the PR 🙂